modules/consensus/accept_test.go

```diff
@@ -650,17 +639,15 @@ func (cst *consensusSetTester) testFileContractsBlocks() error {
 		}},
 	}
 	copy(txn.StorageProofs[0].Segment[:], segment)
-	err = cst.tpool.AcceptTransaction(txn)
+	err = cst.tpool.AcceptTransactionSet([]types.Transaction{txn})
```

Comment:
you know, if you made `AcceptTransactionSet` variadic, then the n=1 case would be much cleaner...guess it depends on how frequent it is.